pontoon/base/urls.py

```diff
@@ -30,6 +30,11 @@
     url(r'^contributor/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
         RedirectView.as_view(url="/contributors/%(email)s/", permanent=True)),
 
+    # API: Toogle user profile attribute
```

Comment:
Django REST Framework is the main choices for Django APIs these days, and we should use it eventually. But for this PR it's not worth it.